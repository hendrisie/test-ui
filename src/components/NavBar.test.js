import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';
describe('NavBar', () => {
  it('whould render correctly', () => {
    const component = shallow(<NavBar />);
    expect(component).toMatchSnapshot();
  });

  it('whould render correctly when title is set', () => {
    const component = shallow(<NavBar title={'Test'} />);
    expect(component).toMatchSnapshot();
  });
});
