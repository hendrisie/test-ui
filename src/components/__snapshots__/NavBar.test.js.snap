// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar whould render correctly 1`] = `ShallowWrapper {}`;

exports[`NavBar whould render correctly when title is set 1`] = `ShallowWrapper {}`;
